account-kit/react/src/components/auth/card/footer/help-text.tsx

```diff
@@ -2,7 +2,8 @@ import { useUiConfig } from "../../../../hooks/useUiConfig.js";
 import { ls } from "../../../../strings.js";
 
 export const HelpText = () => {
-  const { supportUrl } = useUiConfig();
+  const { supportUrl } = useUiConfig(({ supportUrl }) => ({ supportUrl }));
```

Comment:
Can this be
```ts
const supportUrl = useUiConfig(({ supportUrl }) => supportUrl);
```
? Likewise throughout.

Reply:
oh yea I think this was because the initial write of this actually had the generic on `useUiConfig` to be `T extends Record<string, unknown>` which required this syntax. For the most part I left it like this for extensibility, but it's a bit cumbersome to write